syntax = "proto3";

package user;

message UserRequest {
  string jwt_token = 1;

}

message UserResponse {
  string name = 1;
  string email = 2;

}

service User {
  rpc GetUser(UserRequest) returns (UserResponse) {};
}